evalml/tests/component_tests/test_utils.py

```diff
@@ -111,14 +111,10 @@
 )
 not_supported_in_windows_py39 = set(
     [
-        "ARIMA Regressor",
-        "Polynomial Detrender",
         "Prophet Regressor",
     ]
 )
-not_supported_in_linux_py39 = set(
-    ["ARIMA Regressor", "Polynomial Detrender", "Exponential Smoothing Regressor"]
-)
+not_supported_in_linux_py39 = set()
```

Comment:
Should we just get rid of this initialization altogether?

Reply:
Let's keep it in case we need to add to it at a later date but I am adding a comment now.

Reply:
SGTM! Hopefully enough time has passed that most packages have added 3.9 support but it's a good structure to keep as a pattern for what we can do in these exception cases :)